Improve prerequisites section for Windows (#934)
rust-lang/rustc-dev-guide: src/building/prerequisites.md

### Windows

* Install [winget](https://github.com/microsoft/winget-cli)

> `winget` is one of Windows package managers. It will make package installation easy

Reviewer: ```suggestion
`winget` is a Windows package manager. It will make package installation easy
```

on Windows.

Run the following in a terminal:

```powershell
winget install python
winget install cmake
```

If any of those is installed already, winget will detect it.
Then edit your systems `PATH` variable and add: `C:\Program Files\CMake\bin`.

For more information about building on Windows,

> see [the rust-lang/rust's README](https://github.com/rust-lang/rust#building-on-windows).

Reviewer: ```suggestion
see [the `rust-lang/rust` README](https://github.com/rust-lang/rust#building-on-windows).
```
